pywps/app/Service.py

```diff
@@ -381,7 +379,6 @@ def _parse_and_execute(self, process, wps_request, uuid):
                 for proc_outpt in process.outputs:
                     if outpt == proc_outpt.identifier:
                         resp = Response(proc_outpt.data)
-                        resp.call_on_close(process.clean)
```

Comment:
I agree with this change, but I'd also remove the `call_on_close` function entirely, and I'd put that in a separate commit from the one removing `os.chdir(process.workdir)`.

Reply:
interesting - I'll go revisit in a moment